src/Generator/GeneratorTrait.php

```diff
@@ -44,4 +81,22 @@ private function getMeta(array $environments, string $parserFilename): MetaEntry
 
         return $allMetas[$parserFilename];
     }
+
+    /**
+     * @param Environment[] $environments
+     */
```

Comment:
`@param` is out of date

Reply:
even if we expect an array of objects ?
this is usefull for phpstorm to do some completion...